asv/benchmarks/setup/bench_model.py

```
@@ -78,6 +75,42 @@ def teardown(self, robot, num_worlds):
         del self._model
 
 
+class KpiInitializeRender:
+    params = (["g1"], [8192])
+    param_names = ["robot", "num_envs"]
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)